homeassistant/components/octoprint/services.yaml

```diff
@@ -0,0 +1,27 @@
+printer_connect:
+  fields:
+    device_id:
+      required: true
+      selector:
+        device:
+          integration: octoprint
+    profile_name:
+      required: false
+      selector:
+        text:
+    port:
+      required: false
+      selector:
+        text:
+    baudrate:
+      required: false
+      selector:
+        select:
+          options:
+            - "9600"
```

Comment:
Are these the only options octoprint is supporting?

Reply:
Yup